shell/ptty_windows.go

```diff
@@ -11,6 +13,25 @@ import (
 	"github.com/gruntwork-io/terragrunt/options"
 )
 
+func PrepareConsole() {
+	enableVirtualTerminalProcessing(os.Stderr)
+	enableVirtualTerminalProcessing(os.Stdout)
+}
+
+func enableVirtualTerminalProcessing(file *os.File) {
+
+	var mode uint32
+	handle := windows.Handle(file.Fd())
+	if err := windows.GetConsoleMode(handle, &mode); err != nil {
+		fmt.Printf("failed to get console mode: %v\n", err)
+	}
+
+	if err := windows.SetConsoleMode(handle, mode|windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING); err != nil {
+		fmt.Printf("failed to set console mode: %v\n", err)
```

Comment:
Same here with error handling

Reply:
Do we need [this fallback](https://github.com/containerd/console/blob/f652dc3/console_windows.go#L49-L51)?

Reply:
That's good question - Windows docs does not say what happens to the state when error is returned. Just in case I've added setting console mode to previous value